pkg/maps/nat/stats/stats.go

```diff
@@ -41,8 +43,23 @@ type Stats struct {
 	config   Config
 	natMap4  nat.NatMap4
 	natMap6  nat.NatMap6
+
+	Observable4 stream.Observable[TupleCountIterator]
+	next4       func(TupleCountIterator)
```

Comment:
It seems like complete[4,6] is not being called. Is it okay not to call it, e.g. during shutdown?

Reply:
good point, ill clean that up on the shutdown hook.